deps/build.jl

```diff
@@ -1,4 +1,4 @@
-run(`unzip -oj -d ext "MockFtpServer-2.6-bin.zip" "*.jar"`)
+run(`unzip -oj -d ext "MockFtpServer-2.6-bin.zip"`)
```

Comment:
Probably best to leave a comment about why you are doing this. Otherwise I could see this change being reverted and causing some extra work.